apps/docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -58,5 +102,62 @@ const sidebar = SIDEBAR[langCode];
     if (target && target.offsetTop > window.innerHeight - 100) {
       document.querySelector('.nav-groups').scrollTop = target.offsetTop;
     }
+
+    const subTitles = document.querySelectorAll('.subTitle');
+
+    subTitles.forEach((title) =>
+      title.addEventListener('click', (e) => {
+        e.target.nextElementSibling.classList.toggle('hidden');
+        const iconParent = document.querySelector('.caret-icon');
+
+        iconParent.innerHTML = e.target.nextElementSibling.classList.contains(
+          'hidden'
+        )
+          ? `<svg
+          width="24"
+        height="24"
+            xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 24 24"
+      fill="none"
+    >
+      <path
+        d="M12 14L15 10.9999"
+        stroke="white"
+        stroke-width="1.2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      />
+      <path
+        d="M9 11L12 14"
+        stroke="white"
+        stroke-width="1.2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      />
+    </svg>`
```

Comment:
please extract those as variables above, and use them here in ternary

Reply:
Extracting it is a good idea, done